rpc/metrics.go

```diff
@@ -0,0 +1,23 @@
+package rpc
+
+import (
+	prom "github.com/harmony-one/harmony/api/service/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	prom.PromRegistry().MustRegister(
+		rpcRateLimitCounterVec,
+	)
+}
+
+var (
+	rpcRateLimitCounterVec = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "rpc",
```

Comment:
please use 'hmy' as the namespace for all harmony metrics. you can use "rpc" for subsystem.

Reply:
good point! Thanks